pkg/crypto/keys/private_key.go

```diff
@@ -14,21 +14,31 @@ import (
 	"github.com/nspcc-dev/rfc6979"
 )
 
-// PrivateKey represents a NEO private key.
+// PrivateKey represents a NEO private key and provides a high level API around
+// ecdsa.PrivateKey.
 type PrivateKey struct {
-	b []byte
+	ecdsa.PrivateKey
```

Comment:
`type Private ecdsa.PrivateKey`? I mean, this `struct` wrapper seems to be useless now.

Reply:
In this case method `(*keys.PrivateKey).PublicKey()` duplicates field `(ecdsa.PrivateKey).PublicKey`. At the same time we cannot remove the method as we need to have access to `keys.PublicKey` via `keys.PrivateKey` (not to `ecdsa.PublicKey`). So we can either rename `(*keys.PrivateKey).PublicKey()` or let PrivateKey remain a struct. What do you prefer?

Reply:
OK, let it be a struct for now, we can refactor it later.